### PR TITLE
fix: add a default creation date if it is not present

### PR DESCRIPTION
**Pull request by gpoujol**

### CODE REVIEW

File: shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt [hunk 1]
```diff
@@ -304,11 +304,11 @@ object JsonLdUtils {
         } else
             null
 
-    fun getPropertyValueFromMapAsDateTime(values: Map<String, List<Any>>, propertyKey: String): ZonedDateTime? {
+    fun getPropertyValueFromMapAsDateTime(values: Map<String, List<Any>>, propertyKey: String): ZonedDateTime {
```
[the remark this answers is not in the document]
Comment (gpoujol) [reply shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt:307]: It's not here I had the default creation date?

[the remark this answers is not in the document]
Comment (bobeal) [reply shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt:307]: it must be specific to the migration script, you can't change the behavior of a common shared function.

[the remark this answers is not in the document]
Comment (gpoujol) [reply shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt]: Ok I understand I change this

**Review by bobeal — APPROVED** [review]

**Comment by gpoujol** [comment]
> The build has failed...

The build failed cause of that https://github.com/stellio-hub/stellio-context-broker/blob/79620f16b1d1e7d6ea8c0e28ac00c262d734fce0/shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt#L116-L118. I already change it in other PR. Do you want I change here or I let like this ?

**Comment by bobeal** [comment]
> > The build has failed...
> 
> The build failed cause of that
> 
> https://github.com/stellio-hub/stellio-context-broker/blob/79620f16b1d1e7d6ea8c0e28ac00c262d734fce0/shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt#L116-L118
> . I already change it in other PR. Do you want I change here or I let like this ?

OK, let it like this.

**Pull request merged**